docs/core/use-atom.mdx

```diff
@@ -122,7 +117,7 @@ const Counter = () => {
 }
 ```
 
-Similar to the `useSetAtom` hook, `useAtomValue` allows you to access a read-only atom.
+Similar to the `useSetAtom` hook, `useAtomValue` allows you to access a read-only atom. Nonetheless, it can also be used to access read-write atom's values.
```

Comment:
From the past phrase it seemed life you can use `useAtomValue` only with read-only atoms